frontend/src/components/App/Layout.tsx

```diff
@@ -44,11 +54,133 @@ const useStyle = makeStyles(theme => ({
 
 export interface LayoutProps {}
 
+const CLUSTER_FETCH_INTERVAL = 10 * 1000; // ms
+
+interface Config {
+  [prop: string]: any;
+}
+
+function ClusterNotFoundPopup(props: {
+  isClusterNotAvailable: boolean;
+  changeClusterNotAvailableStateHandler: (isClusterNotAvailable: boolean) => void;
+}) {
+  const { isClusterNotAvailable, changeClusterNotAvailableStateHandler } = props;
+  const theme = useTheme();
+  const cluster = getCluster();
+  const { t } = useTranslation('frequent');
+  const isSidebarOpen = useTypedSelector(state => state.ui.sidebar.isSidebarOpen);
+  const history = useHistory();
+
+  if (!cluster) {
+    return null;
+  }
+
+  if (!isClusterNotAvailable) {
+    return null;
+  }
+  return (
+    <>
+      <Box
+        display={'flex'}
+        justifyContent="center"
+        style={{
+          position: 'absolute',
+          color: theme.palette.common.white,
+          textAlign: 'center',
+        }}
+        bgcolor={'error.main'}
+        zIndex={1400}
+        width={'100%'}
+        p={0.5}
+        alignItems="center"
+      >
+        <Box ml={isSidebarOpen ? '-25vw' : 0}>
+          {t(`Something went wrong with cluster ${cluster}`)}
+        </Box>
+        <Box
+          style={{ cursor: 'pointer' }}
+          color={'error.main'}
+          bgcolor={theme.palette.common.white}
+          ml={1}
+          p={0.5}
+          onClick={() => {
+            history.replace('/');
+            changeClusterNotAvailableStateHandler(false);
+          }}
+        >
+          {t('frequent| Take me to cluster chooser')}
```

Comment:
You don't need the `frequent` here, since it's the default namespace when you declared the `useTranslation`. If you keep this, please remove the space after the pipe.

Reply:
updated